validation/genesis/validation-inputs/1135/deploy-config.json

```diff
@@ -16,16 +16,16 @@
   "l2OutputOracleChallenger": "0xBeA2Bc852a160B8547273660E22F4F08C2fa9Bbb",
   "l2GenesisBlockGasLimit": "0x1c9c380",
   "l1BlockTime": 12,
-  "baseFeeVaultRecipient": "0xdA6e5640aFB2ED212Ba3a6fd83076e2ad3daD185",
-  "l1FeeVaultRecipient": "0xdA6e5640aFB2ED212Ba3a6fd83076e2ad3daD185",
-  "sequencerFeeVaultRecipient": "0xdA6e5640aFB2ED212Ba3a6fd83076e2ad3daD185",
-  "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
-  "l1FeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
-  "sequencerFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
-  "baseFeeVaultWithdrawalNetwork": 0,
-  "l1FeeVaultWithdrawalNetwork": 0,
-  "sequencerFeeVaultWithdrawalNetwork": 0,
-  "proxyAdminOwner": "0xdA6e5640aFB2ED212Ba3a6fd83076e2ad3daD185",
```

Comment:
Can you please double check these changes? The automatic genesis validation check `validate-genesis-allocs` uses this data, and the existing values were enough to make it pass, the new values cause it to fail.

Reply:
I have confirmed the changes with our RaaS provider, which were made when ownership of the chain was transferred from the deployer EOA to a multisig. The values were also adjusted (decreased) for fee withdrawals. As a result, the new values are correct.

This was not the case for Lisk Sepolia so genesis validation check passes inside its PR.

Reply:
All those changes were made around 22nd July 2024.

Reply:
I've reverted the changes inside `deploy-config.json` file.